serverless.yaml

```diff
@@ -0,0 +1,36 @@
+service: serverless-example
+
+# username: postgres
+# password: 2dIkIbYqoSwU26Ed8TQO
+
+plugins:
+  - serverless-offline
+  - serverless-dotenv-plugin
```

Comment:
Seems like we don't need this plugin because env variables are now accessible by serverless itself using usedotev: true option. Please, take a look https://www.serverless.com/framework/docs/environment-variables